fuel-core/src/service/graph_api.rs

```diff
@@ -1,6 +1,8 @@
 use super::modules::Modules;
 use crate::database::Database;
 use crate::schema::{build_schema, dap, CoreSchema};
+#[cfg(feature = "prometheus")]
+use crate::service::metrics::metrics;
```

Comment:
I think it might be better to leave the metrics route always configured, but use the feature flags to adjust the API handler response when disabled.

```suggestion
use crate::service::metrics::metrics;
```

Reply:
Ok, will adjust `/metrics` to just return a blank body then if Prometheus is not enabled